playbooks/roles/python-packages/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# tasks file for python-packages
+- name: Install python packages
+  pip:
+    name:
+    - qtpy==1.6.0
```

Comment:
Maybe instead of putting all of the packages here, they would be better of inside the "vars" folder and include them with:
```
include_vars: '{{ vars_file }}'
```
So the role can be generic and reusable. A complete example is here: https://github.com/lerwys/lnls-ans-role-epics-dev/blob/master/tasks/main.yml